ui/components/ui/token-details/token-details.component.js

```diff
@@ -0,0 +1,154 @@
+import React, { useCallback, useContext } from 'react';
```

Comment:
[REQ] This is actually a page, not a component, and should be placed in the pages folder. It should also drop the .component syntax.